apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -0,0 +1,52 @@
+<%
+  var netsimConstants = require('./netsimConstants');
+  var dataConverters = require('./dataConverters');
+
+  var getEncodingLabel = require('./netsimUtils').getEncodingLabel;
+
+  var EncodingType = netsimConstants.EncodingType;
+  var PacketUIColumnType = netsimConstants.PacketUIColumnType;
+
+  var formatAB = dataConverters.formatAB;
+  var formatBinary = dataConverters.formatBinary;
+  var formatHex = dataConverters.formatHex;
+  var alignDecimal = dataConverters.alignDecimal;
+  var binaryToAB = dataConverters.binaryToAB;
+  var binaryToHex = dataConverters.binaryToHex;
+  var binaryToDecimal = dataConverters.binaryToDecimal;
+  var binaryToAscii = dataConverters.binaryToAscii;
```

Comment:
Style change sounds good.  These are all actually used; in some cases I combine a conversion method and a formatting method on one line to get what I want, e.g.
```javascript
dataConverters.alignDecimal(dataConverters.binaryToDecimal(binary, chunkSize))
```